### DISCUSSION TITLE
ImportError: DLL load failed while importing _ifcopenshell_wrapper:

### DISCUSSION THREAD

**Discussion by Borna07** [IfcOpenShell/IfcOpenShell#2064] [Q&A]
Hello!

I am trying to use Ifcopenshell on my work computer and encountered the following error message:

"ImportError: DLL load failed while importing _ifcopenshell_wrapper: %1 ist keine zulässige Win32-Anwendung."

I created a python virutal enviroment and copied ifcopenshell in the site-packages folder like I did on my private computer but something is wrong.

Can anybody help me out?

**Comment by aothms** [comment]
Can you double check that 32/64bit and python version exactly match?

**Reply by Borna07** [reply]
Thanks for the quick reply! I double-checked but it still didn't work...

But I created a new environment on a disk where I have more rights and there it works perfectly! strange but it works

**Reply by aothms** [reply]
Yes unfortunately this is error is very undescriptive. Not a lot we can do. It can definitely have something to do with rights also. Glad you figured it out.
